Gaijinizer patch/Map070.txt

```diff
@@ -497,9 +509,9 @@ so!!Such a person! !#MTLed
 
 その現場を隠れて目撃しちゃったの♡
 > CONTEXT: Map070/events/5/pages/1/177/Dialogue/enemy5
-Occasionally, that Mio-sama is in the middle of
-milking. It's time to get lost.
-I've hidden the site and witnessed it#MTLed
+From time to time, I've slipped unseen into
+Mio-sama's boundary while she's milking. And
+that's when I saw it.#RonIffy
```

Comment:
Note

Reply:
I'm just keeping the word "boundary" for consistency's sake. As she's referring to the boundary that demons put up when they attack people. (as described in 2-3)